web/templates/settings/account_settings.hbs

```diff
@@ -9,9 +9,14 @@
                 <div class="input-group">
                     <label class="settings-field-label" for="change_email_button">{{t "Email" }}</label>
                     <div id="change_email_button_container" class="{{#unless user_can_change_email}}disabled_setting_tooltip{{/unless}}">
```

Comment:
I think we need the class here. This is because of the following state:
- When we open two browser windows and log into the first as normal user and in another as owner.
- Turn on the `Prevent users from changing their email address` setting from the owner window.
- In the updated changes, although the tooltip appears for email changes disabled in the normal user's window for once but if the window of the normal user is refreshed, the tooltip disappears.
- The same is the reason for using `hide` class in the button (`#change_email_button`) as `{{#unless user_can_change_email}}hide{{/unless}}`.

I am including a screen recording to demonstrate this state.

<details>
<summary>Screenrecording of above state.</summary>

https://github.com/user-attachments/assets/b457ba01-ab6d-40e7-ba37-36fd9174ef03

</details>